components/ContactCard/ContactCard.module.scss

```
@@ -7,45 +7,63 @@
 
 .content {
     display: flex;
+
+    .meta {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
 }
 
 .name {
     @include paragraph;
 
-    margin: 0 auto $spacing-half;
+    margin: 0;
 }
 
 .position {
     @include text-small;
 
-    margin: 0 auto $spacing-3;
+    margin: 0;
     color: $color-base-500;
     font-weight: $font-weight-regular;
 }
 
 .links {
     display: flex;
     flex-direction: column;
-    margin: $spacing-2 auto (-$spacing-2);
 }
 
 .linkGroup {
     display: flex;
+    gap: $spacing-3;
     flex-wrap: wrap;
 
     &:not(:last-child) {
-        margin-bottom: $spacing-2;
+        margin-bottom: $spacing-4;
+    }
+
+    &.social {
+        gap: $spacing-3;
+
+        .link {
+            color: $color-base-600;
+        }
+    }
+
+    .signature &:not(.social) {
```

This selector looks incorrect. It would compile to `.linkGroup .signature`, but I guess you wanted `.linkGroup.signature`. Also, the `&:not` selector would apply to children of `.signature` class, not itself

```suggestion
    &.signature:not(.social) {
```

---

It's correct or at least it works the way I want it to. What I wanted is this:
```
.signature .linkGroup:not(.social)
```

In other words - if the parent has `.signature` class (in my case `.container`), apply these rules to all `.linkGroup` elements except if it has `.linkGroup.social`.

---

Oh, I got a bit confused, was thinking this is applied to the container class. Sorry!